spyder/app/mainwindow.py

```diff
@@ -269,6 +270,19 @@ def qt_message_handler(msg_type, msg_log_context, msg_string):
 qInstallMessageHandler(qt_message_handler)
 
 
+class SpyderProxyStyle(QProxyStyle):

```

Comment:
Let's move this one to `spyder/utils/qthelpers.py` because I think it fits better there.

Reply:
Ok will do!